include/z64sram.h

```diff
@@ -0,0 +1,20 @@
+#ifndef Z64SRAM_H
+#define Z64SRAM_H
+
+#include "ultra64/ultratypes.h"
+
+typedef struct {
+    /* 0x00 */ u8* readBuff;
+} SramContext; // size = 0x4
```

Comment:
fyi we have this struct in `z64save.h`

<https://github.com/zeldaret/mm/blob/2080f9906f083a4138d68caa027e4ed56953a169/include/z64save.h#L159>

Not sure what's better tho, if putting it on save.h or being in its own header :man_shrugging:

Reply:
it is definitely more of a lower level structure used for actually fetching the data from sram, as opposed to the stuff that makes up the save context and is more game oriented